yellowbrick/features/importances.py

```diff
@@ -0,0 +1,250 @@
+        numeric description of the feature importance is shown.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+
+    >>> from sklearn.ensemble import GradientBoostingClassifier
+    >>> visualizer = FeatureImportances(GradientBoostingClassifier())
+    >>> visualizer.fit(X, y)
+    >>> visualizer.poof()
+    """
+
+    def __init__(self, model, ax=None, labels=None, relative=True, **kwargs):
+        super(FeatureImportances, self).__init__(model, ax, **kwargs)
+
+        # Data Parameters
+        self.set_params(labels=labels, relative=relative)
+
+    def fit(self, X, y=None, **kwargs):
+        """
+        Fits the estimator to discover the feature importances described by
+        the data, then draws those importances as a bar plot.
+
+        Parameters
+        ----------
+        X : ndarray or DataFrame of shape n x m
+            A matrix of n instances with m features
+
+        y : ndarray or Series of length n
+            An array or series of target or class values
+
+        kwargs : dict
+            Keyword arguments passed to the fit method of the estimator.
+
+        Returns
+        -------
+        self : visualizer
+            The fit method must always return self to support pipelines.
+        """
+        super(FeatureImportances, self).fit(X, y, **kwargs)
+
+        # Get the feature importances from the model
+        self.feature_importances_ = self._find_importances_param()
+
+        # Normalize features relative to the maximum
+        if self.relative:
+            maxv = self.feature_importances_.max()
+            self.feature_importances_ /= maxv
+            self.feature_importances_ *= 100.0
+
+        # Create labels for the feature importances
+        # NOTE: this code is duplicated from MultiFeatureVisualizer
+        if self.labels is None:
+            # Use column names if a dataframe
+            if is_dataframe(X):
+                self.features_ = np.array(X.columns)
+
+            # Otherwise use the column index as the labels
+            else:
+                _, ncols = X.shape
+                self.features_ = np.arange(0, ncols)
+        else:
+            self.features_ = np.array(self.labels)
+
+        # Sort the features and their importances
+        sort_idx = np.argsort(self.feature_importances_)
+        self.features_ = self.features_[sort_idx]
+        self.feature_importances_ = self.feature_importances_[sort_idx]
+
+        # Draw the feature importances
+        self.draw()
+        return self
+
+    def draw(self, **kwargs):
+        """
+        Draws the feature importances as a bar chart; called from fit.
+        """
+        # Quick validation
+        for param in ('feature_importances_', 'features_'):
+            if not hasattr(self, param):
+                raise NotFitted("missing required param '{}'".format(param))
```

Comment:
perhaps we should put this validations in __init__?

Reply:
I thought about it, however, those params won't appear on the estimator until after they're fitted - so we can't do anything until then. 

I wanted some advice on this though - it would be nice if we could take an already fitted model and draw the feature importance without having to run through the data again.  Any thoughts on how to incorporate this here?

Reply:
It would be nice to have some kind of general utility to use in the score and model visualizers that would check to see if the model is fitted already, and if it is, return some useful information (e.g. hyperparam settings, coefficients/feature importances) that could be used by the visualizers.

Reply:
Updated #297 in response to these comments.